doc/src/Packages_details.rst

```diff
@@ -365,27 +365,26 @@ The CG-DNA package requires that also the `MOLECULE <PKG-MOLECULE>`_ and
 
 ----------
 
-.. _PKG-CG-SDK:
+.. _PKG-CG-SPICA:
 
-CG-SDK package
+CG-SPICA package
 ------------------
 
 **Contents:**
 
 Several pair styles and an angle style which implement the
-coarse-grained SDK model of Shinoda, DeVane, and Klein which enables
-simulation of ionic liquids, electrolytes, lipids and charged amino
-acids.
+coarse-grained SPICA (formerly called SDK) model which enables
+simulation of biological or soft material systems.
 
 **Author:** Axel Kohlmeyer (Temple U).
```

Comment:
You should change this to "Original Author" and add an entry "Maintainer" with your and Wataru's affiliation info.

Reply:
I changed the author information as you mentioned.